nautobot/extras/views.py

```diff
@@ -1270,8 +1271,19 @@ def alter_queryset(self, request):
         return queryset
 
     def extra_context(self):
+        # Determine user's preferred view
+        if self.request.GET.get("view") in ["list", "tiles"]:
+            view = self.request.GET.get("view")
+            if self.request.user.is_authenticated:
+                self.request.user.set_config("extras.joblistview.view", view, commit=True)
```

Comment:
Not sure if `extras.joblistview.view` is a good key, looking for confirmation

Reply:
We don't have a real consistent pattern here. The table config form uses `tables.<ClassName>` to store table column selection, while other views in this file use `extras.<model_name>.format` to store JSON vs YAML rendering preference. I do think `display` might be a better leaf key than `view` though. Given that, I'd suggest either `views.JobListView.display` or `extras.job.display` - no strong preference either way.

Reply:
After all I decided to go with `extras.job.display` and overall changed `view` param name to `display`. Thanks for suggestions